Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM 416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-core-runtime:1.0.0
+
+ARG NODE_VERSION=18.12.1
+ARG NPM_VERSION=8.19.2
+
+RUN dnf update -y && \
+    dnf install -y  \
+    git \
+    make \
+    nodejs-$NODE_VERSION \
+    npm-$NPM_VERSION \
+    unzip \
+    zip && \
```

Comment:
We need to ensure the minimal packages are installed. `make` won't be required, for example and it's unlikely that `git` will be required either.